runtime/src/verify.rs

```diff
@@ -9,6 +9,13 @@ use zerocopy::FromBytes;
 /// Handle the `ECDSA384_SIGNATURE_VERIFY` mailbox command
 pub(crate) fn handle_ecdsa_verify(drivers: &mut Drivers, cmd_args: &[u8]) -> CaliptraResult<()> {
     if let Some(cmd) = EcdsaVerifyCmd::read_from(cmd_args) {
+        if !caliptra_common::checksum::verify_checksum(
+            cmd.chksum,
+            CommandId::ECDSA384_VERIFY.into(),
+            &cmd_args[4..],
```

Comment:
Would be good to add a comment explaining that this offset is due to skipping the checksum

Reply:
Done.